content/aips/WSTETH-POLYGON-LISTING.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Add wstETH to Polygon Aave v3
+discussions: https://governance.aave.com/t/arc-add-support-for-wsteth-on-polygon-v3/12266
+shortDescription: Add wstETH to Polygon Aave v3 deployment as collateral, with borrowing enabled and an Ethereum eMode.
+author: Llama - TokenLogic & efecarranza
+created: 2023-04-17
+---
+
+# Summary
+
+This AIP presents the community with the opportunity to add wstETH to the Polygon v3 Liquidity Pool.
+
+# Abstract
+
+Across the Aave v3 Liquidity Pools, wstETH is listed on Ethereum, Arbitrum and Optimism. This proposal seeks to list wstETH on the Polygon Liquidity Pool as Collateral, with Borrowing enabled and an Ethereum eMode.
```

Comment:
might make sense to note that weth is also added to that `correlated` eMode and in specification list how that eMode is configured.